codes/classical/q-ary_digits/reed_solomon.yml

```diff
@@ -22,21 +22,39 @@ features:
   decoders:
```

Comment:
There are 2 more basic decoding algorithms:
-Gorenstein-Peterson-Zierler decoder
-Berlekamp-Massey decoder
(reference: Blahut, Theory and practice of error-control codes, Addison-Wesley 1983)

Reply:
sorry, i dont have any access to this book, do you mean both of them with same reference?

Reply:
You'll likely find the relevant papers just from this info.